src/App.js

```diff
@@ -1,23 +1,25 @@
-import logo from './logo.svg';
 import './App.css';
+import Header from './components/Header';
+import LoggedIn from './components/LoggedIn';
+import WelcomeParagraph from './components/WelcomeParagraph';
+import Nav from './components/Nav';
+import { Routes, Route } from "react-router-dom";
+import Reviews from './components/Reviews';
 
 function App() {
+  const LoggedInUser = "BakriKhalifa"
```

Comment:
Bear in mind, when you get to the ticket that requires a user, that user will need to exist in your back end data set.